pages/common/iperf3.md

```diff
@@ -0,0 +1,23 @@
+# iperf3
+
+> Traffic generator for testing network bandwidth.
+
+- Run iperf3 as a server:
+
+`iperf3 -s`
+
+- Run an iperf3 server on a specific port:
+
+`iperf3 -s -p{{port}}`
+
+- Start bandwidth test:
+
+`iperf3 -c{{server}}`
+
+- Run iperf3 in multiple threads:
```

Comment:
Isn't "threads" the incorrect term for this ? `-P` specifies the no. of connections to make to the server. A single thread can also create multiple connections.

Reply:
Perhaps `multiple parallel streams`?

Reply:
Sounds good to me, @pxgamer. Ping @agnivade

Reply:
Sure, sounds good.